### DISCUSSION TITLE
How to switch out Auto/volume state images

### DISCUSSION THREAD

**Discussion by InterstellarDrifter** [elgatosf/elgato#21] [Ideas]
I purchased a profile package created by sideshowfx(WindowsCommandCenterXL), which has A LOT of cool button icons as well as multi state volume sliders. I was wondering if there was a way to swap out the stock Elgato Auto volume slider images with the new Mackie sliders I purchased. 

I am not a programmer yet, but I'm working on it lol. So bear with me and my limited understanding of some concepts. I might say I'm in the bottom of the dunning-kruger tub graph, lol.

**Comment by ChekTek** [comment]
Hi!

The sliders are built using SVG files and the plugin calculates where to draw the levels. Unfortunately, there is not an easy way to swap in other sliders.

**Reply by InterstellarDrifter** [reply]
Thank you for your reply!
